blocks/tabs/tabs.css

```diff
@@ -111,6 +111,11 @@
     flex: 1 0 60%;
     position: relative;
 
+    .embed-wrapper {
+      width: 100%;
+      height: 100%;
```

Comment:
this height is breaking the tabs carousel, it covers the text and button content.